src/landpatterns/numbering.stanza

```diff
@@ -183,6 +183,16 @@ numbering scheme. For example - a `col-offset = 1` would construct:
     8   12
   ```
 
+@param col-skip Value to divide the raw column value by. This parameter
+is useful when working with staggered grids where a pad is active every
+other column. When this value is greater than 1 - then the numbering scheme
+will accept twice as many columns as normal.
+
+@param row-skip Value to divide the raw row value by. This parameter
+is useful when working with a staggered grid where a pad is active every
+other row. When this value is greater than 1 - then the numbering scheme
+will accept twice as many rows as normal.
+
 @throws ValueError If num-pads is not divisible by num-columns.
 <DOC>
 public defstruct Column-Major-Numbering <: Numbering :
```

Comment:
lol I can't comment on unchanged lines -- what are your thoughts on just requiring a single offset value? I guess you probably have a much better idea of the common use case but when I used this I ended up having to calculate the row/col offsets before passing them in here, when I already just had a hard value offset that I wanted the pad-ids to start at. i.e. I wanted to start the counting at 20 and just provide that as a starting id. 

Just my 2 cents. If you think users are going to use this more from the perspective of offsetting by rows and columns then ignore my babbling

Reply:
> I guess you probably have a much better idea of the common use case but when I used this I ended up having to calculate the row/col offsets before passing them in here, when I already just had a hard value offset that I wanted the pad-ids to start at. i.e. I wanted to start the counting at 20 and just provide that as a starting id.

I guess I don't understand. I needed to do the exact same thing in the ethernet connector and I could just pass `col-offset = 10` or whatever and it did exactly what i needed to do. There is nothing forcing you to pass in a `row-offset` if you don't need to. 

```
    num-scheme = Row-Major-Numbering(2, 1, col-offset = 12)
```

I don't see any reason to remove.